tests/functional/test_install.py

```diff
@@ -271,6 +271,53 @@ def test_install_from_local_directory(script, data):
     assert egg_info_folder in result.files_created, str(result)
 
 
+def test_install_from_relative_directory(script, data):
+    """
+    Test installing requirements using a relative path.
+    """
+    fspkg_folder = script.site_packages / 'fspkg'
+    egg_info_folder = (
+        script.site_packages / 'FSPkg-0.1.dev0-py%s.egg-info' % pyversion
+    )
+    egg_link_file = (
+        script.site_packages / 'FSPkg.egg-link'
+    )
+
+    # Create directories in scratch of varying depth to test relative path to
+    # requirements.
+    for relative_depth in range(3):
```

Comment:
I'd say, no need to test to the third level, 2 seems enough ( the test suite is already quite slow).